packages/api-clients/open-api/m2mGatewayApi.yml

```
@@ -7637,6 +7741,30 @@ components:
         - WAITING_FOR_APPROVAL
         - ARCHIVED
         - REJECTED
+    PurposeDraftUpdateSeed:
+      type: object
+      additionalProperties: false
+      description: Contains the expected payload for purpose update.
+      properties:
+        title:
+          type: string
+          minLength: 5
+          maxLength: 60
+        description:
+          type: string
+          minLength: 10
+          maxLength: 250
+        isFreeOfCharge:
+          type: boolean
+        freeOfChargeReason:
+          type: string
```

Shouldn't this be removable? It's an optional field in the model and on creation
```suggestion
        freeOfChargeReason:
          type: string
          nullable: true
```